docs/configuration/style-modification.md

```diff
@@ -0,0 +1,16 @@
+# Style Modification
+
+## Overview
+
+This document describes how you can apply the styles for the AsyncApi component.
+
+## Changing styles
+
+In the AsyncApi component we use pure css styling. Each component inside root component has a unique css class. Each class has form: `asyncapi__{ELEMENT}--{MODIFIER}`, where:
+
+- `{ELEMENT}` - is a name of specific element. Each element name is the concatenation of the names of the elements in which it is located. For example: `asyncapi__channel-header-title` is located in `header` HTML element of `channel` element. 
```

Comment:
```is located in `header` HTML element``` - is this always true? This may lead to situation in which someone does something like `header.asyncapi__channel-header-title {}`, but in reality there could be div element. Maybe instead of outlining things like that just say we follow BEM and link to it?

Reply:
But here I described that `title` is in `header`, not `*-title` is HTML header.